text/0000-native-cli.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: native_cli
+- Start Date: 2020-09-17
+- RFC PR: (leave this empty)
+- Neon Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes streamlining the directory layout and command-line UX of Neon projects, making the design simpler, more intuitive, and easier to learn and adopt.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC proposes three simplifications to the Neon design:
+
+- **Simplified model:** A Neon project is no longer a JavaScript package that _contains_ a native module; a Neon project _is just_ a native module.
+- **Simplified CLI:** Using Neon no longer requires learning a custom CLI tool; Neon projects are operated with the standard `npm` and `cargo` tools.
+- **Simplified layout:** A Neon project no longer contains a mix of JS and Rust subdirectory structures; it's flattened into a standard Cargo project + a `package.json` to treat it as a Node native module.
+
+This design lightens the cognitive burden of learning Neon, making it less intimidating for new users and easier to use for everyone.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Creating a new Neon project
+
+The `npm init neon` command makes it easy to start a new Neon project. You can use `npm init neon my-app` to create a new `my-app` Neon directory, or `npm init neon` to initialize an existing directory as a Neon project.
```

Comment:
Nit, we _just_ discussed potentially eliminating appending `my-app` on the initial revision. We may want to consider changing this to further expansion.